testdata/p4_16_errors/stack3.p4

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
Where is the output generated by the compiler for this test?
There should be a file stack3.p4-stderr at the very least.